app/components/common-chart.tsx

```diff
@@ -27,6 +30,12 @@ const GenericChart = ({
     queryFilters,
   };
 
+  // Remove any chart errors on chart switch
+  useEffect(
+    () => setChartError("none"),
+    [chartConfig.chartType, setChartError]
+  );
```

Comment:
This effect could be done directly in the hook if chartType is given to it.

Reply:
I changed the approach, so I think this is not relevant atm